### PR TITLE
add a circle config

### PR DESCRIPTION
**Pull request by amandamholl**
for Circle 2.0

### CODE REVIEW

**Comment by arincon9** [comment]
@amandamholl Awesome! Thank you!

**Comment by amandamholl** [comment]
@jasonwc I had to manually do `run: bundle install` for rspec, which I think is because it wasn't persisting the bundled gems. Not sure if you have any ideas about what could be causing that?

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        keys:
+        - v1-dep-{{ .Branch }}-
+        - v1-dep-master-
+        - v1-dep-
+    - run: gem install bundler -v "=1.13.6"
+    - run: mkdir -p tmp/cache
+    - run:
+        name: bundle
+        command: |
+          bundle check --path=~/mavenlink/mavenlink_gem/bundle || bundle install --path=~/mavenlink/mavenlink_gem/bundle --jobs=4 --retry=3
```
Comment (jasonwc) [inline .circleci/config.yml]: I _think_ theres a mismatch here with `mavenlink/mavenlink_gem/bundle` vs `mavenlinkn/mavenlink_gem/current/bundle` thats causing your gem persistence issue. I'd consider removing the idea of `current` from your working directory default and persist to workspace bits to resolve it.

**Comment by jasonwc** [comment]
Flipped the bits @amandamholl

**Comment by amandamholl** [comment]
Thanks for the help @jasonwc!!

**Pull request merged**